apis/python/src/tiledbsoma/_common_nd_array.py

```diff
@@ -88,7 +88,6 @@ def create(
         # the upper limit is exactly 2**63-1.
 
         context = _validate_soma_tiledb_context(context)
-        _util.validate_platform_config(platform_config)
```

Comment:
I'm confused

What I want is to have -- and to verify by unit-test cases -- that:

* If they call `create` with `platform_config={"tiledb": {"create": {key: value, ...}}}` it works
* If they call `create` with `platform_config = TileDBCreateOptions(key=value)` it works
* If they call `create` with _anything else_ we raise an exception right there at runtime

Maybe I'm missing something but I don't see how the third bullet's functionality is isn't lost by this PR